indexers/rewards-squid/src/blocks/utils.ts

```diff
@@ -362,6 +362,59 @@ export async function processRewardEvent(
   return rewardEvent;
 }
 
+export async function updateAccountRewards(
```

Comment:
This same function is in processEvents.ts. Is this intended?

Reply:
You got me worried for a minute that I had added it while it was already there.
But yeah it's in processEvents.ts of the old squid, so we are porting these logics to the rewards-squid :)